### ISSUE TITLE
Accessibility issues -- new website styling

### ISSUE THREAD

**Issue by bmuramatsu** [digitalcredentials/dcc-website#12]
- We should separate style from structure — headings are used for styling more than structure. (This is bad accessibility design (and should not have existed on the WordPress or old site but it does.)
- Above implies styles for the types of text blocks on each page.
- It would be great if the design was accessible from the start. Contrast ratios are not accessible in the menu.

**Comment by fields37** [comment]
Hi @bmuramatsu I'm working on overhauling the styling, but what exactly do you mean by separating style from structure? Do you have any examples of how they are currently not separated?

**Comment by fields37** [comment]
Also, are you suggesting we add an option to adjust contrast ratios in the menu? I think it would be easier to make sure that the website meets the minimum accessible contrast ratios by default, which I believe we did when we first designed the site. I just briefly did some checks and it looks like all the parts of the site meet a AAA contrast ratio.

**Comment by bmuramatsu** [comment]
It looks like the site has been updated to have accessible contrast ratios. Thanks.
